Model/DispatchPlugin.php

```diff
@@ -12,15 +12,15 @@
 
 class DispatchPlugin
 {
-    private $request;
-    private $response;
-    private $lockService;
-    private $resourceConnection;
-    private $modificationTimeRepo;
-    private $errorProcessor;
+    private \Magento\Framework\Webapi\Request $request;
```

Comment:
This seems really verbose if the class is already imported in the `use` statement, also this doesn't match the constructor arg?